src/main.rs

```diff
@@ -13,6 +13,12 @@ fn main() {
     // Ignore errors loading `.env` file.
     let _ = dotenv::dotenv();
 
+    // Ensure it's possible to close Crater with a Ctrl+C even inside Docker (as PID 1).
+    ctrlc::set_handler(|| {
+        std::process::exit(0);
```

Comment:
Maybe use `exit(1)`, as it is not a normal exit?

Reply:
Right.